app/controllers/my_jobs_controller.rb

```diff
@@ -1,5 +1,6 @@
 class MyJobsController < ApplicationController
   def index
     @jobs = current_user.jobs_liked.includes(:company)
+    @tags = Tag.pluck(:name)
```

Comment:
should these tags be scoped to the job? Such as @tags = jobs_liked.tags.pluck(:name)